server/core/src/config.rs

```diff
@@ -47,10 +44,20 @@ pub struct OnlineBackup {
     /// (it's very similar to the standard cron syntax, it just allows to specify the seconds at the beginning and the year at the end)
     pub schedule: String,
     #[serde(default = "default_online_backup_versions")]
-    /// How many past backup versions to keep
+    /// How many past backup versions to keep, defaults to 7
     pub versions: usize,
 }
 
+impl Default for OnlineBackup {
+    fn default() -> Self {
+        OnlineBackup {
+            path: "/dev/null".to_string(),
```

Comment:
Seems like a chaotic default 🤔

I wonder if we could make a backups dir next to the DB or something?L

Reply:
Done. Urgh, messy.